orbit/pkg/update/options.go

```diff
@@ -82,6 +95,21 @@ var (
 		},
 	}
 
+	DesktopLinuxArm64Target = TargetInfo{
+		Platform:             "linux-arm64",
+		Channel:              "stable",
+		TargetFile:           "desktop.tar.gz",
+		ExtractedExecSubPath: []string{"fleet-desktop", constant.DesktopAppExecName},
+		CustomCheckExec: func(execPath string) error {
+			cmd := exec.Command(execPath, "--help")
+			cmd.Env = append(cmd.Env, fmt.Sprintf("LD_LIBRARY_PATH=%s:%s", filepath.Dir(execPath), os.ExpandEnv("$LD_LIBRARY_PATH")))
```

Comment:
The LD_LIBRARY_PATH change will not be needed AFAICS. It's currently only needed for older versions of Fleet Desktop for amd64 where we used a package that required a C library.

Reply:
Removed 👍